index.html

```diff
@@ -13741,8 +13741,11 @@ <h2>Example IDL Attribute Usage</h2>
 
 </section>
 <section class="informative">
-	<h2>Privacy and Security Considerations</h2>
+	<h2>Security Considerations</h2>
 	<p>This specification introduces no new security considerations.</p>
```

Comment:
On second thought, should this paragraph be duplicated in the new/separated Privacy Considerations section?